src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepDynamicContext.java

```diff
@@ -158,8 +180,8 @@ private static abstract class Translator<T> extends DynamicContext.Typed<T> {
         }
 
         @Override protected FilePath get(DelegatedContext context) throws IOException, InterruptedException {
-            if (LOGGER.isLoggable(Level.FINE)) {
-                LOGGER.fine("ESDC=" + context.get(ExecutorStepDynamicContext.class) + " FPR=" + context.get(FilePathDynamicContext.FilePathRepresentation.class));
+            if (LOGGER.isLoggable(Level.FINER)) {
```

Comment:
or use a `Supplier` lambda

Reply:
Awkward in this case because `context.get` throws checked exceptions.

Reply:
:cry: right, nm